bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -316,30 +316,17 @@ void recordStorageCacheBehavior(MetaData metaData) {
 
     private void loadPlugins() {
         NativeInterface.setClient(this);
-        enableOrDisableNdkReporting();
-        enableOrDisableAnrReporting();
-        BugsnagPluginInterface.INSTANCE.loadRegisteredPlugins(this);
```

Comment:
Removing this line breaks the plugin interface.

Reply:
I've updated this as I hadn't considered plugins such as cocos2dx which aren't part of this repository.